File: Machine/ClientApp/src/ViewModel/productView.ts
import {ProductCategory} from "../Enum/ProductCategory";

export class ProductView{
	public id: number;
	public name: string;
	public countProduct: number;
	public price: number;
	public description: string;
	public category: ProductCategory;
	public isRemoved: boolean;
	constructor(id: number,
				name: string,
				countProduct: number,
				price: number,
				description: string,
				category: ProductCategory,
				isRemoved: boolean) {
		this.id = id;
		this.name = name;
		this.countProduct = countProduct;
		this.price = price;
		this.description = description;
		this.category = category;
		this.isRemoved = isRemoved;
	}
}

File: Machine/Models/ViewModel/ProductView.cs
using System;
using Machine.Models.Enum;

namespace Machine.Models.ViewModel
{
	public class ProductView
	{
		public Int32 Id { get; set; }
		public String Name { get; set; }
		public Int32 CountProduct { get; set; }
		public Int32 Price { get; set; }
		public String Description { get; set; }
		public ProductCategory Category { get; set; }
		public Boolean IsRemoved { get; set; }

		public ProductView() { }

		public ProductView(Int32 id, String name, Int32 countProduct, Int32 price, String description, ProductCategory category, Boolean isRemoved)
		{
			Id = id;
			Name = name;
			CountProduct = countProduct;
			Price = price;
			Description = description;
			Category = category;
			IsRemoved = isRemoved;
		}
	}
}